content/integrate/redis-data-integration/architecture.md

```diff
@@ -89,17 +89,18 @@ the backpressure mechanism.
 
 RDI supports the following database sources using [Debezium Server](https://debezium.io/documentation/reference/stable/operations/debezium-server.html) connectors:
 
-| Database                    | Versions               |
-| --------------------------- | ---------------------- |
-| Oracle                      | 12c, 19c, 21c          |
-| MariaDB                     | >= 10.5                |
-| MySQL                       | 5.7, 8.0.x             |
-| Postgres                    | 10, 11, 12, 13, 14, 15 |
-| SQL Server                  | 2017, 2019             |
-| Google Cloud SQL MySQL      | 8.0                    |
-| Google Cloud SQL Postgres   | 15                     |
-| Google Cloud SQL SQL Server | 2019                   |
-| Google Cloud AlloyDB for PostgreSQL | |
+| Database                    | Versions               | Supports Amazon RDS |
```

Comment:
Amazon RDS supports several db types.
We support RDS MySQL, RDS Oracle, RDS SQL Server, RDS Postgres, RDS Mariadb with the same versions we support on prem
(List of supported versions can be updated based on Debezium 2.7 support matrix)

Reply:
https://debezium.io/releases/